dynamic_programming/maximum_circular_subarray.py

```diff
@@ -0,0 +1,48 @@
+# The function returns maximum circular contiguous sum in a[]
+def max_circular_sum(arr: list[int], size: int) -> int:
+    """
+    Input: arr[] = {8, -7, 9, -9, 10, -11, 12}
+    Output: 23
+    Explanation: Subarray 12, 8, -7, 9, -9, 10 gives the maximum sum, that is 23.
+
+    >>> max_circular_sum([8, -7, 9, -9, 10, -11, 12], 7)
+    23
+
+    >>> max_circular_sum([8, -7, 10, -9, 10, -11, 12], 7)
+    24
+    """
+
+    # Edge Case
+    if size == 1:
```

Comment:
add test cases please

Reply:
I had to the add test cases for the edge case right?